OpenRA.Mods.Common/Traits/World/ColorPickerManager.cs

```diff
@@ -179,7 +182,7 @@ Color MakeValid(float hue, float sat, MersenneTwister random, List<(float R, flo
 
 			// Failed to find a solution within a reasonable time: return a random color without any validation
 			onError?.Invoke(InvalidPlayerColor);
-			return Color.FromAhsv(random.NextFloat(), float2.Lerp(HsvSaturationRange[0], HsvSaturationRange[1], random.NextFloat()), V);
+			return Color.FromAhsv(random.NextFloat(), float2.Lerp(HsvSaturationRange[0], HsvSaturationRange[1], random.NextFloat()), random.NextFloat() / 2 + 0.5f);
```

Comment:
Can you add a comment explaining why the `Value` float is `/ 2 + 0.5`?

Reply:
this is the testcase commit